src/view/services/command.manager.js

```diff
@@ -0,0 +1,12 @@
+export class CommandManager {
```

Comment:
Personally I don't like this name:)
For me it seems to be something like execution environment. What about Executor or maybe a Sandbox?